components/proxy/src/main/java/com/hotels/styx/admin/handlers/UptimeHandler.java

```diff
@@ -0,0 +1,50 @@
+/*
+  Copyright (C) 2013-2019 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.admin.handlers;
+
+import com.hotels.styx.api.Eventual;
+import com.hotels.styx.api.HttpInterceptor;
+import com.hotels.styx.api.HttpRequest;
+import com.hotels.styx.api.HttpResponse;
+import com.hotels.styx.api.MetricRegistry;
+import com.hotels.styx.api.WebServiceHandler;
+
+import static com.hotels.styx.api.HttpHeaderNames.CONTENT_TYPE;
+import static com.hotels.styx.api.HttpResponseStatus.OK;
+import static java.lang.String.format;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Provides an uptime via admin interface.
+ */
+public class UptimeHandler implements WebServiceHandler {
+    private final MetricRegistry metricRegistry;
+
+    public UptimeHandler(MetricRegistry metricRegistry) {
+        this.metricRegistry = metricRegistry;
+    }
+
+    @Override
+    public Eventual<HttpResponse> handle(HttpRequest request, HttpInterceptor.Context context) {
+        Object uptime = metricRegistry.getGauges().get("jvm.uptime.formatted").getValue();
+
+        return Eventual.of(HttpResponse.response(OK)
+                .disableCaching()
+                .addHeader(CONTENT_TYPE, "application/json")
+                .body(format("\"%s\"", uptime), UTF_8)
```

Comment:
Is this json content?

Reply:
Hi. @dvlato, @chrisgresty said it is ;-)

Reply:
:)
JSON used to insist that only a map or an array was a valid document, but a few years ago this was changed to allow other value types (string, integer, etc). You could consider using `objectMapper.writeValueAsString(uptime)` instead.

Reply:
I guess technically we can say returning a quoted string is returning valid JSON... :)